app/views/powerbanks/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="card" style="width: 18rem;">
```

Comment:
Possibly looking at changing to container and using card on powerbanks page? Something to amend in the next commit